src/App.tsx

```diff
@@ -1,20 +1,28 @@
-import { Route, Routes } from 'react-router-dom';
+import { Route, Routes, useNavigate } from 'react-router-dom';
 import classes from './App.module.scss';
 import LoginPage from './pages/LoginPage/LoginPage';
 import RegistrationPage from './pages/RegistrationPage/RegistrationPage';
 import Sidebar from './components/Sidebar/Sidebar';
 import { sidebarItems } from './utils/SidebarItems';
+import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import { useEffect, useState } from 'react';
 
 const App = () => {
-  // TODO: after login is finished check if user is logged in (if it isn't remove sidebar and move paths for login and registration)
+  const navigate = useNavigate();
+  const [token, setToken] = useState<string | null>(null);
+
+  useEffect(() => {
+    setToken(localStorage.getItem('token'));
+  }, [navigate]);
+
   return (
     <div className={classes['c-app']}>
-      <Sidebar items={sidebarItems} />
+      {token && <Sidebar items={sidebarItems} />}
```

Comment:
Instead of conditionally rendering sidebar create `<Layout />` component that uses `<Sidebar />` and use that in all protected routes